### PR TITLE
BaSP-MR-203: Response messages corrections

### PR DESCRIPTION
**Pull request by IvanJukonis**
This PR is directly related to ticket MR-203. Changes were made to the messages of all responses to ensure consistency.
Additionally, JEST was disabled due to encountering issues.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "app_name description",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
+    "test": "echo",
```
Comment (NicoCagnina) [inline package.json:7]: Revert this change. Just modify this file:

`.husky/pre-commit`

to this: 
```
#!/bin/sh
. "$(dirname "$0")/_/husky.sh"

npm run lint
```

**Review by NicoCagnina — CHANGES_REQUESTED** [review]

File: src/controllers/class.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const getClasses = (req, res) => {
     .then((data) => {
       if (data) {
         res.status(200).json({
-          message: 'All classes',
+          message: 'Classes list',
           data,
         });
       }
```
Comment (YhanahiRosenthal) [inline src/controllers/class.js:13]: you should add a property "data" and "error", the response it should be message: 'classes list', data: data, error: false

File: src/controllers/class.js [hunk 3]
```diff
@@ -27,7 +27,7 @@ const getClassById = (req, res) => {
     .then((data) => {
       if (data) {
         res.status(200).json({
-          message: 'Class Found',
+          message: 'Class found',
           data,
           error: false,
```
Comment (YhanahiRosenthal) [inline src/controllers/class.js:32]: here is missing property, data: data

File: src/controllers/class.js [hunk 4]
```diff
@@ -70,7 +70,7 @@ const updateClass = (req, res) => {
     .then((result) => {
       if (result) {
         res.status(201).json({
-          message: 'Class Updated',
+          message: 'Class updated',
           result,
           error: false,
         });
```
Comment (YhanahiRosenthal) [inline src/controllers/class.js:76]: here too

File: src/controllers/member.js [hunk 5]
```diff
@@ -70,19 +70,23 @@ const updateMember = (req, res) => {
     .then((result) => {
       if (!result) {
         return res.status(404).json({
-          message: `Member with id: ${id} was not found`,
+          message: 'Member not found',
           error: true,
         });
```
Comment (YhanahiRosenthal) [inline src/controllers/member.js:75]: here, data: null

File: src/controllers/member.js [hunk 6]
```diff
@@ -70,19 +70,23 @@ const updateMember = (req, res) => {
...
       }
-      return res.status(201).json(result);
+      return res.status(201).json({
+        message: 'Member updated',
+        result,
+        error: false,
+      });
     })
```
Comment (YhanahiRosenthal) [inline src/controllers/member.js:82]: here too

File: src/controllers/member.js [hunk 7]
```diff
@@ -117,18 +121,20 @@ const getById = (req, res) => {
...
   try {
     const { id } = req.params;
-
     const memberExist = await Member.findById(id);
-
     if (!memberExist) {
-      return res.status(404).send('ID was not found');
+      return res.status(404).send({
+        message: 'Member not found',
+      });
```
Comment (YhanahiRosenthal) [inline src/controllers/member.js:128]: here

**Review by YhanahiRosenthal — CHANGES_REQUESTED** [review]
i commented something

File: src/controllers/class.js (hunk 2 above)
Comment (IvanJukonis) [reply src/controllers/class.js:13]: The ticket only required the changes on messages, the structure of the http response will be changed in another ticket.

**Review by YhanahiRosenthal — APPROVED** [review]
Good job!

File: package.json (hunk 1 above)
Comment (IvanJukonis) [reply package.json]: Done!!!!

**Comment by IvanJukonis** [comment]
Response structure on components Member and Classes were not fixed because they are related to another TICKET.

**Review by NicoCagnina — APPROVED** [review]

**Review by YhanahiRosenthal — APPROVED** [review]
Good job!

**Review by CristianLotorto — APPROVED** [review]
Great work!

**Comment by GleViDe** [comment]
2nd Testing Results

Tested locally on https://nico-megarocket-server.vercel.app/
Test result: Pass

Great job!!
